src/feature/button/Button.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import buttonStyle from "./Button.module.css";
+
+interface ButtonProps {
+  children?: React.ReactNode;
+  onClick?: () => void;
+}
+
+const Button = ({ children }: ButtonProps, { onClick }: ButtonProps) => {
```

Comment:
Можно ведь так сделать:
```suggestion
const Button = ({ children, onClick }: ButtonProps) => {
```